### ISSUE TITLE
Not working on git bash terminal

### ISSUE THREAD

**Issue by hyorman** [hymkor/expect#12]
I am using VsCode for my development. And my default terminal is git bash in VsCode. But when I run expect.exe on real git bash, it could not read the output or it could not send any command after spawn. Here is my screenshot and code. It waits forever. But however it runs perfectly when i run it on vscode teminal. Despite VsCode also uses the git bash. First screenshot is vscode terminal which is success.

![vscode](https://user-images.githubusercontent.com/49436369/59777384-3fe5a680-92bd-11e9-8a8e-18de404f0452.PNG)

FAILED SCREEN IN BELOW

![Capture](https://user-images.githubusercontent.com/49436369/59777501-77ece980-92bd-11e9-8f1e-f6061845794a.PNG)

HERE IS MY CODE

`echo(true)
if spawn([[c:\Program Files\Git\usr\bin\bash.exe]],"ssh-copy-id","-i",arg[1],arg[2],arg[3]) then
    expect("?")
    echo(false)
    sendln("yes")
    expect("password:")
    sendln("toor")
end`

**Comment by hymkor** [comment]
Thank you for reporting issue.

Is your terminal (not shell) compatible with Windows Command Prompt or are you using a terminal like mintty?
Tell me that the corner's icon of the terminal is like this or not ?

![image](https://user-images.githubusercontent.com/3752189/59883127-0d3bcb00-93ef-11e9-82aa-bb8566fc267f.png)

**Comment by hyorman** [comment]
You are welcome!
Thanks for your interest. 
No I am not using windows command prompt. Here is my git-bash terminal icon.
![image](https://user-images.githubusercontent.com/49436369/59901772-f0aa8d80-9404-11e9-9826-e1fa7b57a1de.png)

**Comment by hymkor** [comment]
I understood.

My `expect.exe` reads text by ReadConsoleOutputW API. But, NOT ON Windows Command Prompt, it can not work as we hope. I have to improve my `expect.exe` to read them like the UNIX version of `expect`. It seems to take a long time.

Would you like to wait ?
（or you can look for the MSYS version of `expect`. )

**Comment by hyorman** [comment]
I have already working on windows, but for generate a linux tool , I am using bash terminal to use unix commands. For now I found a workaround. I am calling windows command prompt and running batch file that calling expect.exe in bash. It seems working now. There is no urgency for me. Thanks for your developing and interest. 

Edit: I also saw expect.exe using syscall windows api. But that is the confusing issue, when i run git-bash as executable expect.exe does not work. But when I run git-bash in vscode, expect.exe runs perfectly. How it can be possible?
